test/odyssey/test_hgram.c

```diff
@@ -0,0 +1,91 @@
+#include <machinarium.h>
+#include <odyssey_test.h>
+#include <assert.h>
+#include <lrand48.h>
+#include "sources/hgram.h"
+
+void hgram_forward_test();
+
+void hgram_backward_test();
+
+int hgram_random_test();
+
+static int myround(double x)
```

Comment:
http://www.cplusplus.com/reference/cmath/round/ round have the same meaning

Reply:
Currently, we do not link with math library. It was common policy for a while. Eventually, we will change that, but not in this PR.